packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss

```diff
@@ -0,0 +1,172 @@
+$red-60v: #b51d09;
+$red-70v: #8b1303;
+
+$small-desktop-screen: 1008px;
```

Comment:
This token also seems like it should not be part of any individual component.

Reply:
I thought about it and couldn't find a better solution for tokens than what Alex did here. These values just don't exist anywhere else that we can use right now, until the remainder of the USWDS tokens are brought into the variables file.

Reply:
Right now we have a tokens file in the [web-components directory](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/_variables.scss), in the [CSS Library](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/dist/tokens/scss/variables.scss), and in [Formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/9578cd69766b7b75dce415188b73f90c79eb85ea/packages/formation/sass/base/_b-variables.scss#L132). Now we are adding random tokens here in a specific component.

It just feels like there are tokens everywhere and adding more here might make dealing with them later more challenging than it already will be.

Can we consider adding the colors we need to the CSS-Library, regenerate the file, and then updating the tokens file in the web-components directory with the new compiled file?